.github/workflows/publish.yml

```diff
@@ -5,22 +5,37 @@ on:
 
 jobs:
     build:
+        name: Build, Sign and Publish
         runs-on: ubuntu-latest
         permissions: write-all
         steps:
-            - uses: actions/checkout@v2
+            - uses: actions/checkout@v3
               with:
                   fetch-depth: 0
             - name: Set up JDK 8
-              uses: actions/setup-java@v2
+              uses: actions/setup-java@v3
               with:
                   java-version: "8"
-                  distribution: "adopt"
+                  distribution: "temurin"
                   server-id: "ossrh"
```

Comment:
is it required to define the `server-id`? we had never use it in the other pipelines we had in teamserver

Reply:
I think so because it's a oss server... maybe it's used only for OSS projects. The earlier version of this file already have it that's the reason I keep it.